health-services/project-factory/src/server/utils/microplanUtils.ts

```
@@ -126,9 +126,9 @@
     }
   }
   if (duplicates.length > 0) {
-    request.body.ResourceDetails.status = resourceDataStatuses.invalid
+    requestBody.ResourceDetails.status = resourceDataStatuses.invalid
   }
-  request.body.sheetErrorDetails = request?.body?.sheetErrorDetails ? [...request?.body?.sheetErrorDetails, ...duplicates] : duplicates;
+  requestBody.sheetErrorDetails = requestBody?.sheetErrorDetails ? [...requestBody?.sheetErrorDetails, ...duplicates] : duplicates;
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)